--- PokerTH-1.0.1-src/src/net/common/ircthread.cpp	2013-04-07 05:04:27.006209660 -0700
+++ PokerTH-1.0.1-src/src/net/common/ircthread.cpp.new	2013-07-02 16:08:09.153632418 -0700
@@ -32,7 +32,7 @@
 #include <net/socket_helper.h>
 #include <net/ircthread.h>
 #include <net/socket_msg.h>
-#include <libircclient/libircclient.h>
+#include <libircclient.h>
 
 // We need to do the following to handle different versions of libircclient.
 // Sadly, libircclient doesn't have actual definitions for its versions in its headers.
